src/api/commons/queries/names.query.js

```diff
@@ -7,13 +7,14 @@ export default [
       _id: 0,
       id: 1,
       resourceId: 1,
-      officialName: 1,
+      officialName: { $ifNull: ['$usualName', null] },
```

Comment:
officialName is required at creation and non nullable at patch time so it should always be there. If not, response will throw error 500 which is what I would expect.

Reply:
Fixed